### ISSUE TITLE
NVDA alpha: Remote access cannot connect

### ISSUE THREAD

**Issue by wmhn1872265132** [nvaccess/nvda#19076]
### Steps to reproduce
- Start NVDA
- Use local host as server connection for remote access

### Actual behavior
Remote access cannot connect

### Expected behavior
Remote access normal connection

### NVDA logs, crash dumps and other attachments
```
IO - speech.speech.speak (10:21:01.818) - MainThread (20908):
Speaking ['日志片段开始点已标记，再按一次复制到剪贴板']
DEBUG - utils.networking._fetchUrlAndUpdateRootCertificates (10:21:02.110) - _checkForUpdatableAddons (18264):
Got response with status code: 200
DEBUG - gui.addonStoreGui.controls.messageDialogs.UpdatableAddonsDialog._checkForUpdatableAddons (10:21:02.150) - _checkForUpdatableAddons (18264):
no updatable add-ons found
IO - inputCore.InputManager.executeGesture (10:21:03.059) - winInputHook (12420):
Input: kb(laptop):enter
INFO - _remoteClient.server.RemoteCertificateManager.ensureValidCertExists (10:21:03.074) - MainThread (20908):
Checking certificate validity
ERROR - unhandled exception (10:21:03.115) - MainThread (20908):
Traceback (most recent call last):
  File "wx\core.pyc", line 3427, in <lambda>
  File "gui\__init__.pyc", line 946, in run
  File "_remoteClient\client.pyc", line 352, in handleDialogCompletion
  File "_remoteClient\client.pyc", line 497, in startControlServer
  File "_remoteClient\server.pyc", line 275, in __init__
  File "_remoteClient\server.pyc", line 297, in createServerSocket
  File "truststore\_api.pyc", line 130, in wrap_socket
  File "truststore\_api.pyc", line 338, in _verify_peercerts
  File "truststore\_api.pyc", line 311, in _get_unverified_chain_bytes
  File "ssl.pyc", line 1176, in get_unverified_chain
AttributeError: 'NoneType' object has no attribute 'get_unverified_chain'
IO - speech.speech.speak (10:21:03.168) - MainThread (20908):
Speaking ['桌面', '列表', '可多选', CancellableSpeech (still valid)]
IO - speech.speech.speak (10:21:03.169) - MainThread (20908):
Speaking ['此电脑', '未选择', '，4 之 1 项', CancellableSpeech (still valid)]
IO - inputCore.InputManager.executeGesture (10:21:05.010) - winInputHook (12420):
Input: kb(laptop):NVDA+control+shift+f1
```

#### NVDA installed/portable/running from source
portable

#### NVDA version
alpha-53001,2fa36585 (2026.1.0.53001)

#### Windows version
Windows 10 21H2 (AMD64) build 19044.6332

#### Name and version of other software in use when reproducing the issue
n/a

#### Other information about your system
n/a

#### Does the issue still occur after restarting your computer?
No

#### Have you tried any other versions of NVDA? If so, please report their behaviors
2025.3 Working normally

#### If NVDA add-ons are disabled, is your problem still occurring?
Yes

#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
Not relevant

**Comment by seanbudd** [comment]
Probably regression from #18528
